api/tests/routes/backoffice/pro_users_test.py

```diff
@@ -41,8 +45,8 @@ class GetProUserTest(GetEndpointHelper):
     endpoint_kwargs = {"user_id": 1}
     needed_permission = perm_models.Permissions.READ_PRO_ENTITY
 
-    # session + current user + pro user data
-    expected_num_queries = 3
+    # session + current user + pro user data + features flag
```

Comment:
```suggestion
    # session + current user + pro user data + feature flag
```
Ou `feature flags`, ça dépend